src/test/PrimaryButton/primaryButton.test.ts

```diff
@@ -0,0 +1,71 @@
+import { afterEach, describe, expect, it } from 'vitest';
+import { cleanup, fireEvent, render, screen } from '@testing-library/svelte';
+
+import type { PrimaryButtonInputs } from '../../lib/types/primaryButton/PrimaryButton';
+import PrimaryButton from '../../components/primaryButton/PrimaryButton.svelte';
+import Spinner from '../../lib/images/Spinner.svelte';
+import CheckMark from '../../lib/images/Vector.svelte';
+
+describe('Primary button', () => {
+    afterEach(() => cleanup());
+
+    const exampleOnClickFunction = async () => {
+        await setTimeout(() => {
+            console.log('pretend to load');
+        }, 2000);
+        return 1 + 1;
+    };
```

Comment:
curious 
- why a 2 second wait, seems long, maybe I don't understand the code
- why return 1+1

Reply:
So 2 second wait is just so the spinner will display but I could shorten it and its just a dummy function to pass in as an example prop so the return doesn't matter. So I just put 1 + 1 just to have something there